person.rb

```diff
@@ -0,0 +1,20 @@
+class Person
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @age = age
+    @name = name
+    @parent_permission = parent_permission
+  end
+
+  attr_writer :id
+  attr_accessor :name, :age
+
+  def of_age?
+    @age >= 18
+  end
+
+  def can_use_services?
+    @parent_permission == true && @age >= 18
+  end
```

Comment:
- [ ]  Good job to create can_use_services method, but you still  have a room of improvement on it. The requirement said that, ![image](https://github.com/joyapisi/school_library/assets/43877689/5072d083-1464-43ff-a553-f7b2e1ebdcde)
but according to your codes, you've used and operators, which wil change the logic of the method. Fix your result to achieve the expected result.